fenix/app/src/main/java/org/mozilla/fenix/search/SearchDialogController.kt

```diff
@@ -158,18 +156,7 @@ class SearchDialogController(
     }
 
     override fun handleTextChanged(text: String) {
-        // Display the search shortcuts on each entry of the search fragment (see #5308)
-        val textMatchesCurrentUrl = fragmentStore.state.url == text
-        val textMatchesCurrentSearch = fragmentStore.state.searchTerms == text
-
         fragmentStore.dispatch(SearchFragmentAction.UpdateQuery(text))
-        fragmentStore.dispatch(
-            SearchFragmentAction.ShowSearchShortcutEnginePicker(
-                !settings.showUnifiedSearchFeature &&
```

Comment:
Note: This dispatch action is always gonna be `false` because of `!settings.showUnifiedSearchFeature`.